doc/integrator/api.rst

```diff
@@ -6,4 +6,18 @@ JavaScript API
 Any c2cgeoportal application exposes a JavaScript API. This API can be used in third-party applications,
 like CMS applications.
 
-.. TODO: fill for new API
+In the folder ``geoportal/<package>_geoportal/static-ngeo/api/apihelp/`` we have all the files related to
+the API help.
+
+In the ``geoportal/demo_geoportal/static-ngeo/api/api.css`` file you have the API CSS.
+
+In the ``geoportal/demo_geoportal/static-ngeo/api/index.js`` wou can customise you API with:
```

Comment:
In the file ``geoportal/demo_geoportal/static-ngeo/api/index.js``, you can customize your API with: